.github/workflows/ci-azure.yml

```diff
@@ -0,0 +1,124 @@
+name: CI Azure
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      ci_backend:
+        description: 'Name of the job backend/target'
+        default: ''
+        required: false
+        type: string
+      ci_option:
+        description: 'Name of the job option for display'
+        default: ''
+        required: false
+        type: string
+      bootstrap_args:
+        default: ''
+        required: false
+        type: string
+      matrix_image:
+        default: ''
+        required: true
+        type: string
```

Comment:
```suggestion
```

Let's remove that as well. I don't think there is a reason to run this test on multiple platforms at least for now.